### ISSUE TITLE
problem with Intangible mod - creating Well of Souls

### ISSUE THREAD

**Issue by hunter688** [SpongePowered/SpongeForge#537]
When use spongeforge , cannot create "Well of Souls"  multiblock structure - portal not created. Without SpongeForge everything works as expected.

http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/wip-mods/2567717-intangible-0-0-21-alpha

**Comment by Aaron1011** [comment]
@hunter688: What are the steps to create a 'Well of Souls'?

**Comment by hunter688** [comment]
place benign stones to ground:

 bbb
b   b
b   b
b   b
 bbb

**Comment by hunter688** [comment]
looks like mod bug, sometimes it does not works...

**Comment by Aaron1011** [comment]
This issue appears to be due to a bug with the current cause tracking system. Once @gabizou's refactor is done, I expect that this issue will be resolved.

**Comment by hunter688** [comment]
I think it is duplicate of #535

**Comment by kinggoesgaming** [comment]
@hunter688 you should leave this open, as there is a possibility that they are not related. Wait till that is resolved before closing this one...

**Comment by hunter688** [comment]
@kinggoesgaming it is resolved as #535

Now it works without problems.

**Comment by kinggoesgaming** [comment]
@hunter688 cheers :clap:
